src/sentry/snuba/referrer.py

```diff
@@ -546,8 +546,24 @@ class ReferrerBase(Enum):
     {f"TSDB_MODELID_{model.value}": f"tsdb-modelid:{model.value}" for model in TSDBModel},
 )
 
+# specific suffixes that apply to tsdb-modelid:4 referrers, these are optional
+# and are passed around through using `referrer_suffix`
+TSDB_4_SUFFIXES = {
+    "user_count_snoozes",
+    "alert_event_frequency",
+    "alert_event_uniq_user_frequency",
+    "alert_event_frequency_percent",
+}
+
+TSDBModel4Referrer = enum.Enum(
```

Comment:
@evanh lemme kno if this looks like a reasonable way to add these referrer suffixes in